packages/fx-core/src/common/template-utils/templatesActions.ts

```diff
@@ -156,6 +160,8 @@ export const fetchTemplateZipFromLocalAction: ScaffoldAction = {
       return;
     }
 
+    context.fallback = true;
```

Comment:
If it is fallback to local template, maybe this action is already tracked in the error handling of download from remote?

Reply:
Seems not - errors will be muted (do nothing) in error handling of "download from remote", see code snippets below:
![image](https://user-images.githubusercontent.com/1803943/181710120-084a7fd1-4dcb-4ca8-8a29-7e917d250b36.png)